core-bundle/contao/dca/tl_content.php

```diff
@@ -1480,6 +1487,23 @@ public function disableButton($row, $href, $label, $title, $icon, $attributes)
 		return System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_ELEMENT_TYPE, $row['type']) ? '<a href="' . $this->addToUrl($href . '&amp;id=' . $row['id']) . '" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . Image::getHtml($icon, $label) . '</a> ' : Image::getHtml(str_replace('.svg', '--disabled.svg', $icon)) . ' ';
 	}
 
+	/**
+	 * Return the edit children button
+	 *
+	 * @param array  $row
+	 * @param string $href
+	 * @param string $label
+	 * @param string $title
+	 * @param string $icon
+	 * @param string $attributes
+	 *
+	 * @return string
+	 */
+	public function editChildren($row, $href, $label, $title, $icon, $attributes)
+	{
+		return System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_ELEMENT_TYPE, $row['type']) ? '<a href="' . $this->addToUrl($href . '&amp;id=' . $row['id']) . '" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . Image::getHtml($icon, $label) . '</a> ' : Image::getHtml(preg_replace('/\.svg$/i', '_.svg', $icon)) . ' ';
```

Comment:
We need to check if the content element supports nesting here. Any idea how to accomplish that?

Reply:
It would work like this: 1f4b3daa04b11968d0b2f9f3bf39a0c9d29b449e

But if that is correct, the method should not be called `isNested()` but `supportsNesting()` or something alike, shouldn‘t it?